endpoints/events/vtrack_test.go

```diff
@@ -29,15 +29,17 @@ const (
 
 // Mock pbs cache client
 type vtrackMockCacheClient struct {
-	Fail  bool
-	Error error
-	Uuids []string
+	Fail   bool
+	Error  error
+	Uuids  []string
+	Values []prebid_cache_client.Cacheable
```

Comment:
added values to assert adapter name from cache client input. This will help to ensure that /vtrack endpoint works with case insensitive adapter name